### PR TITLE
Sds 788 button docs

### PR DESCRIPTION
**Pull request by hsbacot**
+ Prelim docs of `Button` from `@meetup/swarm-components`
+ Added support for `react-docgen` to parse comments and convert them to info used to document the components
+ Added propstable.jsx, a component to render the generated props info in the mdx files
+ removed `retoggle` to resolve ssr issues (temporary, but prefer stable build until this is resolved)

### CODE REVIEW

File: packages/swarm-components/lib/Button.js [hunk 1]
```diff
@@ -73,4 +73,102 @@ const Button = props => {
...
+        "name": "boolean"
+      },
+      "description": "The bordered button"
+    },
+    "className": {
+      "required": false,
+      "flowType": {
+        "name": "string"
+      },
+      "description": "Classes for adidtional styles to be applied"
```
Comment (jaclynj) [inline packages/swarm-components/lib/Button.js]:
```suggestion
      "description": "Classes for additional styles to be applied"
```

File: packages/swarm-components/lib/Checkbox.js [hunk 2]
```diff
@@ -34,4 +34,39 @@ const Checkbox = props => {
...
+      "required": true,
+      "flowType": {
+        "name": "boolean"
+      },
+      "description": ""
+    },
+    "disabled": {
+      "required": false,
+      "flowType": {
+        "name": "boolean"
```
Comment (jaclynj) [inline packages/swarm-components/lib/Checkbox.js]: I'm confused - name is a boolean?

File: packages/swarm-components/src/Button.jsx [hunk 3]
```diff
@@ -4,15 +4,45 @@ import cx from 'classnames';
 import Icon from './Icon';
 
 type Props = {
+	/**
+	* The bordered button
+	*/
 	bordered?: boolean,
+	/**
+	* Classes for adidtional styles to be applied
```
Comment (jaclynj) [inline packages/swarm-components/src/Button.jsx]:
```suggestion
	* Classes for additional styles to be applied
```

File: packages/swarm-components/lib/Checkbox.js (hunk 2 above)
Comment (jaclynj) [reply packages/swarm-components/lib/Checkbox.js]: I don't really understand how these objects for the components work, does it have something to do with `react-docgen` ?

Comment (hsbacot) [reply packages/swarm-components/lib/Checkbox.js]: Yes, it's part of `react-docgen` to be used in the props sections of the pages.

Comment (jaclynj) [reply packages/swarm-components/lib/Checkbox.js]: Could you add something about react-docgen in the readme with a link to their documentation?

**Pull request merged**